WYSIWYG-Quill/wysiwyg.html

```diff
@@ -0,0 +1,186 @@
+    body {
+      margin: 0;
+    }
+
+    div.wysiwyg-box {
+      width: 100%;
+      padding: 0;
+      z-index: 1;
+    }
+
+    .wysiwyg.static {
+      display: inline-block !important;
+      position: static !important;
+      top: 0 !important;
+      left: 0 !important;
+    }
+
+    .disabled_overlay {
+      position: fixed;
+      background-color: #777;
+      z-index: 10;
+      cursor: not-allowed;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      opacity: 0.1;
+    }
+  </style>
+</head>
+
+<body>
+  <!-- Custom element HTML -->
+  <div class="disabled_overlay"></div>
+  <div class="wysiwyg-box">
+    <section id="wysiwyg">
+
+      <div id="standalone-container">
+        <div id="toolbar-container">
+          <span class="ql-formats">
+            <select class="ql-size"></select>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-bold"></button>
+            <button class="ql-italic"></button>
+            <button class="ql-underline"></button>
+            <button class="ql-strike"></button>
+          </span>
+          <span class="ql-formats">
+            <select class="ql-color"></select>
+            <select class="ql-background"></select>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-script" value="sub"></button>
+            <button class="ql-script" value="super"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-header" value="1"></button>
+            <button class="ql-header" value="2"></button>
+            <button class="ql-blockquote"></button>
+            <button class="ql-code-block"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-list" value="ordered"></button>
+            <button class="ql-list" value="bullet"></button>
+            <button class="ql-indent" value="-1"></button>
+            <button class="ql-indent" value="+1"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-direction" value="rtl"></button>
+            <select class="ql-align"></select>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-link"></button>
+            <button class="ql-image"></button>
+            <button class="ql-video"></button>
+            <button class="ql-formula"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-clean"></button>
+          </span>
+        </div>
+        <div id="editor-container"></div>
+      </div>
+
+    </section>
+  </div>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.js"></script>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+
+  <script src="https://amend.cz/wysiwyg/quill.min.js"></script>
```

Comment:
Please use the official source defined in [documentation](https://quilljs.com/docs/download/)

```suggestion
  <script src="//cdn.quilljs.com/1.3.6/quill.min.js"></script>
```